CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing to nightwatch-commands
+
+## Bugs, Feature Requests
+Submit an [issue](https://github.com/mobify/nightwatch-commands/issues).
+
+## Pull Requests
+
+1. Make a new branch from `develop`.
+1. Do your work.
+1. Update the README. 
```

Comment:
Our tests don't currently work. Should we require tests now, or add the requirement after we fix the tests?